api/v1beta1/keystoneapi_types.go

```diff
@@ -83,6 +83,10 @@ type KeystoneAPISpec struct {
 	// Keystone Container Image URL (will be set to environmental default if empty)
 	ContainerImage string `json:"containerImage"`
 
+	// +kubebuilder:validation:Optional
+	// List of Secret resource containing access credentials to the registry.
+	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
```

Comment:
This implementation follows the existing one in rabbitmq-cluster operator. I agree this may not be a very clean implementation so we might want our own interface design though we still have to consider how we manage rabbitmq cluster.